state_machines-activerecord.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'state_machines-activemodel', '>= 0.5.0'
-  spec.add_dependency 'activerecord' , '>= 4.1', '< 5.2'
+  spec.add_dependency 'activerecord' , '>= 4.1', '< 5.3'
```

Comment:
Next major version is going to be `6.0` so we can probably set the upper-bound limit on it